openqa-investigate

```diff
@@ -66,9 +66,8 @@ clone() {
 
     # output: { "$id": $clone_id }
     clone_id=$(echo "$out" | runjq -r ".\"$id\"")
-    url="$scheme://$host/t$clone_id"
     # Create markdown list entry
-    echo "* **$name**: $url"
+    echo "* *$name*: t#$clone_id"
```

Comment:
can you explain this change?

Reply:
openQA will automatically expand `t#123` to a link to that test.

Reply:
Ok. I suggest you put that into a separate explicit commit if it's not too much effort. I am asking that because for me it would take like 2-3m with `git gui` in an interactive rebase :)